### ISSUE TITLE
These strings are hardcoded and cannot be translated

### ISSUE THREAD

**Issue by DAnn2012** [Dan0sz/host-analytics-locally#9]
Hello @Dan0sz 

These strings are hardcoded and cannot be translated:

https://github.com/Dan0sz/host-analytics-locally/blob/831a3b2fc73fe37fe220e01aaa65133c6b512b25/includes/admin/class-settings.php#L36-L54

Thanks.

**Comment by DAnn2012** [comment]
Hi @Dan0sz 

please could you kindly take a look at this my Issue report?

Thank you.

**Comment by Dan0sz** [comment]
It's on my todo list. Thanks for notifying me!

**Comment by DAnn2012** [comment]
Hi @Dan0sz 

I would like to point out that those strings are hardcoded even in the latest version **4.8.2**.

Thanks.

**Comment by Dan0sz** [comment]
Yes, quite a few refactors are required (as it's a constant, containing the strings) to make them translatable. And I haven't gotten around to that yet. Feel free to make a PR!
